### PR TITLE
Convert token id to hexadecimal for collectible transfer

### LINKED ISSUE #2423: Send NFT collectibles will fail when add it with a decimal number of tokenId.
Send NFT collectibles will fail when add it with a decimal number of tokenId.

I add a NFT collectibles with tokenId: 120

It send failed when I try to send it to another account.

But I successed in transfer it in myetherwallet with ABI.

I get the transaction data from the failed tx and successed tx.

success data:

0x23b872dd0000000000000000000000004cf0a877e906dead748a41ae7da8c220e4247d9e0000000000000000000000005560af0f46d00fcea88627a9df7a4798b1b109610000000000000000000000000000000000000000000000000000000000000078

failed data:

0x23b872dd0000000000000000000000004cf0a877e906dead748a41ae7da8c220e4247d9e0000000000000000000000005560af0f46d00fcea88627a9df7a4798b1b109610000000000000000000000000000000000000000000000000000000000000120

At the end of the data, I see 78 and 120.

Apparently, Metamask Mobile treated my decimal Tokenid 120 as hexadecimal. It leads to failure.

Please fix it, thanks.

### PR DESCRIPTION
**Pull request by andrepimenta**
**Description**

Fixes collectibles transfer by converting the token id to hexadecimal before the transfer.

**Issue**

Resolves #2423

### CODE REVIEW

File: app/util/number.js [hunk 1]
```diff
@@ -600,3 +600,12 @@ export const conversionUtil = (
 		invertConversionRate,
 		value: value || '0'
 	});
+
+export const toHexadecimal = decimal => {
```
Comment (sethkfman) [inline app/util/number.js:604]: What do you think about adding a test to this utility function?

**Comment by mobularay** [comment]
@rickycodes to continue this work while @andrepimenta is away on vacation (responding to @sethkfman's question of adding a test).

File: app/util/number.js (hunk 1 above)
Comment (rickycodes) [reply app/util/number.js:604]: test added!

**Review by sethkfman — APPROVED** [review]
LGTM

**Pull request merged**